doc/releases/release-notes-3.7.rst

```diff
@@ -353,6 +353,11 @@ Networking
 
   * Removed IPSP support. ``CONFIG_NET_L2_BT`` does not exist anymore.
 
+* TCP:
+
+  * ISN generation uses SHA256 instead of MD5. Moreover it relies on PSA APIs
+    instead of legacy Mbed TLS functions for hash computation.
+
```

Comment:
Wondering whether this should rather be in the migration guide as the previous Kconfig option behavior (which used to `select MBEDTLS` and others) is changed?

Reply:
Actually the `migration-guide` has an entry for this change as well [here](https://github.com/zephyrproject-rtos/zephyr/pull/73508/files#diff-f80404b1ec33e5720fc31f6c388acee451f54148e7fff15cd55ed94b3546f1cfR554). Based on your comment, I think I can extend it to mention the Kconfig change though